--- boot_patch.sh.orig	2023-07-22 09:37:46.032330080 +0200
+++ boot_patch.sh	2023-07-22 09:38:41.118318842 +0200
@@ -183,6 +183,8 @@
 "$SKIP32 add 0644 overlay.d/sbin/magisk32.xz magisk32.xz" \
 "$SKIP64 add 0644 overlay.d/sbin/magisk64.xz magisk64.xz" \
 "add 0644 overlay.d/sbin/stub.xz stub.xz" \
+"add 0755 overlay.d/init.adb.rc init.adb.rc" \
+"add 0755 overlay.d/sbin/rootshell.sh rootshell.sh" \
 "patch" \
 "backup ramdisk.cpio.orig" \
 "mkdir 000 .backup" \
